README.md

```diff
@@ -1,24 +1,270 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- PROJECT SHIELDS -->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
 
-Things you may want to cover:
 
-* Ruby version
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/jcjurado3/civic_voice_plus_fe">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
 
-* System dependencies
+<h3 align="center">CivicVoice+</h3>
 
-* Configuration
+  <p align="center">
+    Welcome to the front end repository for CivicVoicePlus!
+    CivicVoicePlus is a web application built for the Turing School of Software and Design's Mod 3 Consultancy project. Read more about project requirements: https://backend.turing.edu/module3/projects/consultancy/
```

Comment:
hmm I think CivicVoice+ looks better but people could be mistaken and think its CivicVoice positive